samples/bluetooth/mesh/light/CMakeLists.txt

```diff
@@ -18,6 +18,10 @@ target_sources(app PRIVATE
 	src/model_handler.c)
 target_include_directories(app PRIVATE include)
 
+# Preinitialization related to DFU over SMP for nRF52 series
+target_sources_ifdef(CONFIG_BT_MESH_LIGHT_SAMPLE_NRF52_DFU app PRIVATE
```

Comment:
It requires additional configuration. Can we skip that platform for now just not to complicate this task any more? I guess the support for 52 series is what the customers mainly are interested in

Reply:
Excluding compilation of `smp_dfu.c` by `CONFIG_BOARD_THINGY53_NRF5340_CPUAPP`/`_NS` maybe a good option. I was also thinking about taking another sample, which doesn't support thingy53, but they are not much in demand.

Reply:
Using another sample would solve all problems, except that it would be hard for users to discover it unless we slap the feature in their face through documentation page. Using one of the light related samples is most likely to make users aware of this feature.